pubsub/kafka/sarama_log_bridge.go

```diff
@@ -0,0 +1,21 @@
+package kafka
```

Comment:
Why need to package logger here?

Reply:
This allows someone to run Dapr in debug mode and get the Sarama logs for troubleshooting. It isn't required for the core work in this PR, but I needed the functionality to troubleshoot the development and thought it would be useful to leave it for end-users.